qiskit/transpiler/passes/layout/vf2_utils.py

```diff
@@ -37,7 +37,7 @@ class MultiQEncountered(Exception):
 
     def _visit(dag, weight, wire_map):
         for node in dag.op_nodes(include_directives=False):
-            if isinstance(node.op, ControlFlowOp):
+            if node.is_control_flow():
                 if isinstance(node.op, ForLoopOp):
```

Comment:
You could avoid the op usage here too if you did:
```suggestion
                if node.name == "for_loop":
```